repository/src/main/java/org/apache/atlas/repository/graph/GraphBackedSearchIndexer.java

```diff
@@ -330,6 +330,11 @@ private void initialize(AtlasGraph graph) throws RepositoryException, IndexExcep
             HashMap<String, HashMap<String, Object>> KEYWORD_MULTIFIELD = new HashMap<>();
             KEYWORD_MULTIFIELD.put("keyword", ES_KEYWORD_FIELD);
 
+            HashMap<String, Object> ES_KEYWORD_WO_NORMALIZER = new HashMap<>();
+            ES_KEYWORD_WO_NORMALIZER.put("type", "keyword");
+            HashMap<String, HashMap<String, Object>> KEYWORD_FIELD = new HashMap<>();
+            KEYWORD_FIELD.put("keyword", ES_KEYWORD_WO_NORMALIZER);
```

Comment:
Could have just used KEYWORD_MULTIFIELD? Intead of creating new one? NOT A BLOCKER!

Reply:
KEYWORD_MULTIFIELD also adds an analyser by itself, hence we decided to create one without that. Analysing/supporting different cases won't be needed here, and it could anyway be an overhead.